### PR TITLE
Fix and re-enable pa11y-ci

### PR DESCRIPTION
**Pull request by danswick**
Closes https://github.com/GSA-TTS/FAC/issues/2056

To verify, review the [check logs](https://github.com/GSA-TTS/FAC/actions/runs/6127575033/job/16633551247?pr=2058#step:5:76) for this action OR create a branch off of `ds-workflow-node` and trigger the a11y tests.

Don't mind the unreasonably long list of commits. 

## PR checklist: submitters

- [x]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.
- [x]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [x]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   ~~Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.~~
- [x]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker-clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.
        
The larger the PR, the stricter we should be about these points.

### CODE REVIEW

**Comment by danswick** [comment]
This old-ish Docker issue provided the fix. https://github.com/pa11y/pa11y-ci/issues/198

File: .github/workflows/testing-from-build.yml [hunk 1]
```diff
@@ -53,40 +53,31 @@ jobs:
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Start services
+        working-directory: ./backend
+        run: |
+          touch .env
+          docker compose -f docker-compose.yml up -d
+      
+      - name: Run Lighthouse CI and pa11y
```
Comment (danswick) [inline .github/workflows/testing-from-build.yml:78]: Combined these two together to keep the npm install bits in one place.

File: .pa11yci [hunk 2]
```diff
@@ -7,11 +7,15 @@
...
     "standard": "WCAG2AA",
     "runners": [
       "htmlcs",
       "axe"
     ],
-    "hideElements": ".pa11y-ignore, .usa-select, .usa-step-indicator__segment-label, .is-hidden, #upload-worksheet, .usa-file-input__instructions, #my-submissions"
+    "hideElements": ".pa11y-ignore, .usa-select, .usa-step-indicator__segment-label, .is-hidden, #upload-worksheet, .usa-file-input__instructions, #my-submissions",
+    "chromeLaunchConfig": {
+      "executablePath": "/usr/bin/google-chrome"
+    }
```
Comment (danswick) [inline .pa11yci:19]: After a whole bunch of tests, it turned out pa11y was silently failing to launch headless chrome, then timing out. I couldn't pinpoint the exact source of this change in the dependency tree, but it turns out we just needed to point it to chrome's path.

Ref: https://github.com/pa11y/pa11y-ci/issues/198

**Review by jadudm — APPROVED** [review]
LGTM. Will watch the GH env after merge.

**Pull request merged**